src/metabase/pulse/render/table.clj

```diff
@@ -62,13 +86,31 @@
     (when bar-width
       [:th {:style (style/style (bar-td-style) (bar-th-style) {:width (str bar-width "%")})}])]])
 
+(defn- render-bar
+  [bar-width normalized-zero]
+  (if (< bar-width normalized-zero)
+    (list
+     [:td {:style (style/style (bar-td-style) {:width :99% :border-right "1px solid black" :padding-right 0})}
```

Comment:
style note: it's a bit more readable to separate the kv pairs with commas https://guide.clojure.style/#opt-commas-in-map-literals